src/main/webapp/script.js

```diff
@@ -23,20 +23,37 @@ function getComments() {
       // Display the maximum number of comments
       for (let i = 0; i < data.length; i++) {
           // Format i comments in a list and display to the webpage.
-          if (data[i] != "") {
-              comments.appendChild(createListElement(data[i]));
-            }
+          if(i%3 == 0)
+          {
+            // Bold only the names with each comment
+            comments.appendChild(createBoldElement(data[i]));
+          }
+          else if (data[i] != "") {
```

Comment:
Formatting, would prefer using a string function to check if it's an empty string too.